docs/conf.py

```diff
@@ -129,8 +117,9 @@ def remove_dir_content(path: str) -> None:
 html_static_path = ["_static"]
 html_css_files = ["theme-deltares.css"]
 html_theme_options = {
-    "show_nav_level": 2,
+    "show_nav_level": 1,
```

Comment:
Maybe this is why you don't see the collapsing menus? Looking at core no... might be the one below then? "collapse_navigation"

<img width="1117" height="378" alt="Image" src="https://github.com/user-attachments/assets/5d676dc3-f832-43bb-82b6-32db5489a42a" />